sycl/test-e2e/Graph/README.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+The "Graph" directory contains tests for the sycl_ext_oneapi_graph extension.
+
```

Comment:
This should probably be a `##`-level section by itself - Mention that there should be a `lit.local.cfg` file in any test sub-directories that sets `config.required_features` to the [graph aspect](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#73-querying-device-support) required to run the tests:
* `aspect-ext_oneapi_limited_graph` for any test that doesn't require executable graph update.
* `aspect-ext_oneapi_graph` for any test that does require executable graph update.